### DISCUSSION TITLE
Cannot install runtime update

### DISCUSSION THREAD

**Discussion by mruin** [elementary/appcenter#1982] [Q&A]
Hi! I've just installed elementary os 7 but i'm experiencing an issue with the app center, i cannot install a runtime update.
I've tried to do "flatpak update" and i get this error:

flatpak update
Looking for updates…

        ID                                             Branch            Op            Remote                 Download
 1. [✗] org.freedesktop.Platform.GL.default            22.08             u             freedesktop            49,6 MB / 137,7 MB

Error: Error pulling from repo: While pulling runtime/org.freedesktop.Platform.GL.default/x86_64/22.08 from remote freedesktop: Validating dirtree 6e340b9cffb37a989ca544e6bb780a2c78901d3fb33738768511a30617afa01d (/files/etc/debuginfod/): Dir 0 in dirtree: Invalid NULL filename
Updates complete.
error: There were one or more errors

How can I solve this? Thanks

**Comment by andremorro** [comment]
Same here. Already tried 

```
flatpak repair
flatpak repair --user
```

nothing worked so far

**Comment by dmitrykvasnikov** [comment]
Same problem on MacBook Air mid2012.

**Comment by meisenzahl** [comment]
This should be fixed by https://github.com/elementary/appcenter/pull/1983.

**Reply by bdpor** [reply]
I have made a fresh install elementaryOS 7 keeping my old /home partition which was created and  used with elementaryOS 6.1. 
I have the same error reported above. Followed the link in your post #1983 and was unable to understand whether or not a sollution for the issue has already been implemented. I have updated elementaryOS 7 after install, rebooted and still find the flatpak update error (both in appcenter and CLI). In CLI the output is:

flatpak update
Looking for updates…


        ID                                             Branch            Op            Remote                 Download
 1. [✗] org.freedesktop.Platform.GL.default            22.08             u             freedesktop            11,1 MB / 137,7 MB

Error: While pulling runtime/org.freedesktop.Platform.GL.default/x86_64/22.08 from remote freedesktop: Validating dirtree 6e340b9cffb37a989ca544e6bb780a2c78901d3fb33738768511a30617afa01d (/files/etc/debuginfod/): Dir 0 in dirtree: Invalid NULL filename
Updates complete.
error: There were one or more errors


Thank you for any insight.
